README.md

```diff
@@ -4,7 +4,7 @@ Internet of Good Things ([IoGT](https://www.unicef.org/innovation/IoGT)) is deve
 
 The development uses the Python programming language, building on the [Django](https://www.djangoproject.com/) server-side web framework combined with the [Wagtail](https://wagtail.io/) content management system.
 
-Inline with the latest Wagtail [Long Term Support release](https://github.com/wagtail/wagtail/wiki/Release-schedule), IoGT 2.x supports:
+In line with the latest Wagtail [Long Term Support release](https://github.com/wagtail/wagtail/wiki/Release-schedule), IoGT 2.x supports:
```

Comment:
Could change my markdown links so that they use the same style as yours (which will keep line lenghts shorter)?